packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -248,9 +249,8 @@ export class RulesetExecutor {
               if (this.rulesEngine.debugMode) {
                 output.evaluation = handleRuleEvaluationDebug(rule, this.ruleset.name, output.actions, output.error, runtimeFactValues, factValues, oldFactValues);
               } else if (output.error) {
-                // eslint-disable-next-line no-console
-                console.error(output.error);
-                console.warn(`Skipping rule ${rule.id}, and the associated ruleset`);
+                (this.rulesEngine.logger || console).error(output.error);
```

Comment:
Seeing this, maybe it would make sense to default the rules engine logger to `console`. What do you think?

Reply:
I know, 
What I am doing here is not the best because the logger is a LoggerService which is not the same type as the console (so I can't default on the console). 
I can't default to a new service with the consoleLoggerService client as the service will be injected in the app module.
I could eventually change the type of the logger to reflect something similar to what I have done in the ama-sdk. 

What do you think?